carbon-design-system/carbon
# feat(Pagination, PaginationNav): migrate v11 versions to use new IconButton

Ref #10717

#### Changelog

- updates v11 `Pagination` to use `IconButton`
- updates v11 `unstable_Pagination` to use `IconButton`
- updates v11 `PaginationNav` to use `IconButton`

#### Testing / Reviewing

- Make sure all tests pass
- Make sure components look and work as expected
- Make sure tooltip is showing up on hover
